### ISSUE TITLE
Switch Adrift 4 online player to Parchment

### ISSUE THREAD

**Issue by curiousdannii** [iftechfoundation/ifdb#1223]
Consensus seems to be that the play.adrift.co player is considerably worse than Parchment. Indeed, someone has set up manual Parchment links for almost all Adrift 4 games. It would be better to just switch the default online player so that we can get rid of the manual links.

**Comment by dfabulich** [comment]
Are you sure about this? https://ifdb.org/viewgame?id=clpc8d5s70z8v63j is Adrift 4, and it links to https://ifarchive.org/if-archive/games/competition2003/adrift/sophie/sophie.taf

But https://iplayif.com/?story=https%3A%2F%2Fifarchive.org%2Fif-archive%2Fgames%2Fcompetition2003%2Fadrift%2Fsophie%2Fsophie.taf fails with an error.

> Glk SCARE Error
> 
> Unable to load an Adrift game from the requested file.

@kpeamon Should we switch to Parchment in bulk? Do you know of a workaround for issues like these?

**Comment by kpeamon** [comment]
**On average** I would say that for ADRIFT games **earlier** than version 5 (3.8, 3.9 and 4) plays a lot better than the WebRunner (play.adrift.co is the WebRunner) but none of the interpreters are perfect for Adrift 4 games (and earlier versions).  There are many examples where the WebRunner appears to work well but the parser is very broken for ADRIFT 4 games.

BUT:
1. More or less all **new** ADRIFT games are made with ADRIFT 5, where Parchment does not work and WebRunner is decent
2. I have already set up html links to Parchment for many ADRIFT 4 games (I left out Sophie's Adventure etc because it did not work with Parchment).
3. No one is making ADRIFT 4 games these days, only ADRIFT 5 (could change though..)
4. I'd rather have no play online option than give the player the impression that the game is bugged when the problem is actually the interpreter.

 My opinion for now:
Change nothing but perhaps(?) reconsider when Parchment plays ADRIFT 5 games well. But we should always be able to override by making manual links to the preferred interpreter like I did for a lot of ADRIFT 3.8-4 games. It may always be game dependent which interpreter to use.

**Comment by curiousdannii** [comment]
Yeah there's going to be an even more difficult question of whether to preference the WebRunner or FrankenDrift when I eventually get the latter to work online 😆

**Comment by dfabulich** [comment]
Based on what I'm reading here, I think I'm not going to switch the online player just yet.

I've filed a Parchment bug for Sophie's Adventure. https://github.com/curiousdannii/parchment/issues/161

If @curiousdannii gives the word, I'm willing to try a round of testing to make sure a bunch of top-rated ADRIFT 4 games basically work, and then switch over.
